test/e2e/api-specs/transform.ts

```diff
@@ -0,0 +1,334 @@
+        {
+          name: 'address',
+          value: account,
+        },
+      ],
+      result: {
+        name: 'getEncryptionPublicKeyResult',
+        value: '0x1a8819e0c9bab700',
+      },
+    },
+  ];
+
+  const getTransactionCount = openrpcDocument.methods.find(
+    (m) => (m as MethodObject).name === 'eth_getTransactionCount',
+  );
+  (getTransactionCount as MethodObject).examples = [
+    {
+      name: 'getTransactionCountExampleEarliest',
+      description: 'Example of a pending getTransactionCount request',
+      params: [
+        {
+          name: 'address',
+          value: account,
+        },
+        {
+          name: 'tag',
+          value: 'earliest',
+        },
+      ],
+      result: {
+        name: 'getTransactionCountResult',
+        value: '0x0',
+      },
+    },
+    {
+      name: 'getTransactionCountExampleFinalized',
+      description: 'Example of a pending getTransactionCount request',
+      params: [
+        {
+          name: 'address',
+          value: account,
+        },
+        {
+          name: 'tag',
+          value: 'finalized',
+        },
+      ],
+      result: {
+        name: 'getTransactionCountResult',
+        value: '0x0',
+      },
+    },
+    {
+      name: 'getTransactionCountExampleSafe',
+      description: 'Example of a pending getTransactionCount request',
+      params: [
+        {
+          name: 'address',
+          value: account,
+        },
+        {
+          name: 'tag',
+          value: 'safe',
+        },
+      ],
+      result: {
+        name: 'getTransactionCountResult',
+        value: '0x0',
+      },
+    },
+    {
+      name: 'getTransactionCountExample',
+      description: 'Example of a getTransactionCount request',
+      params: [
+        {
+          name: 'address',
+          value: account,
+        },
+        {
+          name: 'tag',
+          value: 'latest',
+        },
+      ],
+      result: {
+        name: 'getTransactionCountResult',
+        value: '0x0',
+      },
+    },
+    // returns a number right now. see here: https://github.com/MetaMask/metamask-extension/pull/14822
+    // {
+    //   name: 'getTransactionCountExamplePending',
+    //   description: 'Example of a pending getTransactionCount request',
+    //   params: [
+    //     {
+    //       name: 'address',
+    //       value: account,
+    //     },
+    //     {
+    //       name: 'tag',
+    //       value: 'pending',
+    //     },
+    //   ],
+    //   result: {
+    //     name: 'getTransactionCountResult',
+    //     value: '0x0',
+    //   },
+    // },
+  ];
+  // TODO: move these to a "Confirmation" tag in api-specs
+  const methodsWithConfirmations = [
+    'wallet_requestPermissions',
+    'eth_requestAccounts',
+    'wallet_watchAsset',
+    'personal_sign', // requires permissions for eth_accounts
+    'wallet_addEthereumChain',
+    'eth_signTypedData_v4', // requires permissions for eth_accounts
+    'wallet_switchEthereumChain',
```

Comment:
TODO: @shanejonas

Reply:
i think we gotta leave this TODO, still need to figure out the tagging structure for the different things we need it for: `eip155:<chainid>`, confirmations, etc.